Lib/test/test_sqlite3/test_dbapi.py

```diff
@@ -57,6 +57,22 @@ def test_api_level(self):
         self.assertEqual(sqlite.apilevel, "2.0",
                          "apilevel is %s, should be 2.0" % sqlite.apilevel)
 
+    def test_version(self):
+        with self.assertWarns(DeprecationWarning):
+            sqlite.version
```

Comment:
Please also check from _where_ the warnings are emitted. Take a look at #24008 for inspiration :)

Reply:
I missed this comment earlier. Just realized and pushed the change. I checked other tests and felt like using self.assertEqual with `__file__` and cm.filename looked good. Do let me know if this works. Else, i'll change

Reply:
Also, can you please tell me what is full-form of `cm` ? I checked other tests and all of them assign warnings as `cm`.

Reply:
"context manager", if I had to guess.

A